src/core/notifier/handler/webhook/image_delete_handler.go

```diff
@@ -0,0 +1,24 @@
+package webhook
+
```

Comment:
Seems `DeleteImagePreprocessHandler `  `PullImagePreprocessHandler ` `PushImagePreprocessHandler ` are doing the same thing?

Reply:
> Seems `DeleteImagePreprocessHandler ` `PullImagePreprocessHandler ` `PushImagePreprocessHandler ` are doing the same thing?

yes, but the hook type(eg. pullImage, pushImage, deleteImage) is different.

Reply:
it can be part of the event data?

Reply:
> it can be part of the event data?

we considered this， if put hook type into event data, we have to use `switch...case` or other judgement to distinguish hook type.